packages/suite/src/components/suite/PrerequisitesGuide/components/DeviceDisconnectRequired.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
If this is a replacement of [DisconnectDevice modal](https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/components/suite/modals/DisconnectDevice/index.tsx ), the modal should be deleted.

Reply:
removed in 0362fe06c605ef45791af69b2ecf3b93326d33d1